src/OpenApi/Serializer/OpenApiNormalizer.php

```diff
@@ -56,7 +56,7 @@ private function recursiveClean($data): array
             }
 
             // Side effect of using getPaths(): Paths which itself contains the array
-            if ('paths' === $key) {
+            if ('paths' === $key && isset($data['paths']['paths'])) {
                 $value = $data['paths'] = $data['paths']['paths'];
```

Comment:
```suggestion
                $value = $data['paths'] = $data['paths']['paths'] ?? [];
```

Reply:
This is not working, we still have the array type error from the `getPaths` method, i just made a new proposal :)